packages/server/database/migrations/20221021163318-rename-christmas-template.ts

```diff
@@ -0,0 +1,17 @@
+import {R} from 'rethinkdb-ts'
+
+export const up = async function (r: R) {
+  await r
+    .table('MeetingTemplate')
+    .get('aChristmasCarolRetrospectiveTemplate')
+    .update({name: 'Christmas Retrospective'})
```

Comment:
+1 I would assume we want to keep Santa, I asked clarification question in Slack.

Reply:
Yes, confirmed, we want to keep Santa.

Reply:
Thanks, Santa's back!